docs_source_files/content/webdriver/driver_requirements.es.md

```diff
@@ -424,13 +462,20 @@ System.setProperty("webdriver.ie.driver", "C:/path/to/IEDriver.exe")
   {{< / code-panel >}}
 {{< / code-tab >}}
 
-Microsoft tambien ofrece un binario para [Internet Explorer 11 en Windows 7 y 8.1](//www.microsoft.com/en-gb/download/details.aspx?id=44069). El cual no ha sido actualizado desde 2014 y está basado en una versión preeliminar de la especificación del W3C.
-[Jim Evans](//jimevansmusic.blogspot.co.uk/2014/09/using-internet-explorer-webdriver.html) hizo un informe excelente sobre la implementación del controlador.
+Microsoft tambien ofrece un binario para [Internet Explorer 11 en Windows 
+7 y 8.1](//www.microsoft.com/en-gb/download/details.aspx?id=44069).
+ El cual no ha sido actualizado desde 2014 y está basado en una versión preeliminar 
+ de la especificación del W3C.
+[Jim Evans](//jimevansmusic.blogspot.co.uk/2014/09/using-internet-explorer-webdriver.html) 
+hizo un informe excelente sobre la implementación del controlador.
 
 
 ### Opera
 
-Las versiones actuales de Opera están construidas con el motor de Chromium, y Webdriver ahora es soportado [a través del controlador de código propietario de Opera]((//github.com/operasoftware/operachromiumdriver/releases)) el cual puede ser [añadido al PATH](#adding-executables-to-your-path) o configurado como propiedad del sistema.
+Las versiones actuales de Opera están construidas con el motor de Chromium, y 
+WebDriver ahora es soportado [a través del controlador de código propietario 
+de Opera]((//github.com/operasoftware/operachromiumdriver/releases)) el cual puede 
+ser [añadido al PATH](#adding-executables-to-your-path) o configurado como propiedad del sistema.
```

Comment:
Hi @jmartinezpoq,

'#adding-executables-to-your-path navigates' to "Adding executables to your path" in the same page in en document. In your case it should be '#añadiendo-los-ejecutables-al-path-del-sistema'

Reply:
Good catch @Harsha509, @jmartinezpoq, please have a look, thanks!